modin/engines/base/frame/data.py

```diff
@@ -2102,6 +2102,10 @@ def to_pandas(self):
 
         return df
 
+    def wait_computations(self):
```

Comment:
What would you think if we made this a decorator instead? Then decorate all of the methods that trigger computation. 

If it is implemented this way, we could add a config for `modin.config.BenmarkMode` and check within the decorator if that flag is set. What do you think?

Reply:
I suggest leaving both the decorator and the function, since in different situations it is convenient to use either one or the other.

Reply:
It seems that we should cast map object to list

Reply:
I don't think we need both. In fact, I prefer this functionality not be exposed here. We can move the decorator into the frame manager.

I prefer these be deleted here because we don't want others to be accidentally calling them. Benchmark Mode is for internal and developer use only.